docker/llm/serving/cpu/kubernetes/README.md

```diff
@@ -31,6 +31,10 @@ Set hyper-threading to off, ensure that only physical cores are used during depl
 
 The entrypoint of the image will try to set `OMP_NUM_THREADS` to the correct number by reading configs from the `runtime`.  However, this only happens correctly if the `core-binding` feature is enabled.  If not, please set environment variable `OMP_NUM_THREADS` manually in the yaml file.
 
+### Vllm usage
+
+If the worker is expected to run in vllm mode, you should set the args `-w vllm_worker` in worker part of deployment.yaml. 
```

Comment:
If you want to use the vllm `AsyncLLMEngine` for serving, you should set the args `-w vllm_worker` in worker part of `deployment.yaml`.